### DISCUSSION TITLE
Bisher kein auslesen eines Wertes möglich, Bleibt ständig beim Bedienen der WebUI hängen und verliert die Verbindung. Anpingen aber jederzeit erfolgreich?!?!?? Not any reading yet and permanent hang ups and connection loss via WebUI, Ping works fine?!??

### DISCUSSION THREAD

**Discussion by timosust** [jomjol/AI-on-the-edge-device#1296] [Q&A] [answered]
English translated with deepl below. I can read and understand English well, but I don't have the nerve to write in detail right now. Sorry

Moin,

so langsam bin ich verzweifelt. Ich versuche jetzt seit Tagen das Teil zum Laufen zu bekommen und weis einfach nicht mehr weiter.

Ich habe diesen ESP32-Cam https://arduino-projekte.info/produkt/esp32-cam-v2-integriertem-ch340-mit-ov2640-kamera-modul/

Geflasht mittels Flash Download Tool. Hat auch alles soweit geklappt. Der Inhalt des Ordners SDCard auf die SD Karte, wifi.ini entsprechend editieren etc. 

Dann konnte ich auch zunächst auf die Web Oberfläche zugreifen und das Setup machen. Allerdings nicht vollständig, weil er sich immer wieder aufhing. 

Nun gut, WLan Probleme im Keller mit Repeater gelöst, dabei dann ersten ESP gehimmelt.

Neuer ESP wieder entsprechend geflasht etc. Jetzt habe ich das Setup soweit hinbekommen, aber merkte dabei Spannungsprobleme > die Micro USB Buchse hatte sich gelöst. War dann auch gleich ganz ab. Also Spannungsversorgung mittels USB Netzteil u aufgetrenntem Kabel direkt an den +5VVDC Pin und einen GND Pin.

Soweit lief er nun stabil und ich konnte das Setup abschließen. Allerdings zeigt er nun folgendes Verhalten, wo ich nicht weiter komme:

1. Im "Overview" habe ich bisher weder ein Bild noch irgendwelche Werte gesehen.

2. Unter "Recognition" wird entweder ebenfalls nix angezeigt, oder es kommen ein paar komplett verschwommene Ziffern unten in der Leiste. Einmal hatte ich sogar ein Bild vom Zähler, was aber komplett schwarz war und nur die ROI Kästchen blau umrandet angezeigt wurden.

3. Die Weboberfläche scheint sich nach kurzer Zeit aufzuhängen. Besonders im Menü Recognition, aber auch bei Einstellungen etc. Ich finde da bisher kein Muster. 

4. Auch ist die Weboberfläche komischerweise immer nur relativ kurz nachdem ich den ESP über Spannungsunterbrechung neu gebootet habe per IP über Browser (FireFox) aufrufbar. Später, oder wenn ich sie nach hängen bleiben neu laden möchte, bekomme ich immer gemeldet, dass die IP nicht erreichbar sei. >>> Aber, ich kann den ESP JEDERZEIT erfolgreich anpingen?!?? Das dürfte doch dann eigentlich auch nicht möglich sein, oder?!??

OK, heute kann ich anscheinend gar nicht über die webUI erreichen. Auch nicht nach Reset per Spannungsunterbrechung. Pingen geht problemlos, aber im Browser wird entweder Zeitüberschreitung oder Verbindung wurde vom Server zurückgesetzt angezeigt.

Ich kann leider nicht im Menü der FritzBox sehen, ob der ESP vorhanden ist, da der testweise Zusatzrepeater keiner von AVM ist, sondern von Allnet. In dessen Menü wird aber die MAC Adresse angezeigt.

Kennt jemand zufälig dieses Verhalten? Jemand ne Idee, was ich falsch mache? 

Vielen Dank (und ich werde jetzt nochmal in den Keller gehen, die ausgerichtete Kamera wieder abbauen um die SD Karte u die Log Datei auszulesen und hier zu posten)

Timo

Hello,

I'm getting desperate. I've been trying to get this thing to work for days now and I just don't know what to do.

I have this ESP32-Cam https://arduino-projekte.info/produkt/esp32-cam-v2-integriertem-ch340-mit-ov2640-kamera-modul/

I downloaded it with the Flash Download Tool. Everything has worked so far. The contents of the SDCard folder on the SD card, edit wifi.ini accordingly etc.. 

Then I was able to access the web interface and do the setup. However, not completely because it kept hanging up. 

Well, I solved the WLan problems in the basement with a repeater, but then the first ESP got corrupted.

New ESP flashed again accordingly, etc. Now I have managed the setup so far, but noticed voltage problems > the micro USB socket had come loose. It was then also completely off. So I connected the power supply via USB power supply and a disconnected cable directly to the +5VDC pin and a GND pin.

So far it ran stable and I was able to complete the setup. However, it now shows the following behaviour, where I can't get any further. 1:

1. in the "Overview" I have seen neither a picture nor any values so far.

2. either nothing is displayed under "Recognition", or there are a few completely blurred digits at the bottom of the bar. Once I even had a picture of the counter, but it was completely black and only the ROI boxes were displayed with a blue border.

3. the web interface seems to hang up after a short time. Especially in the menu Recognition, but also in settings etc.. So far I can't find a pattern. 

4. strangely, the web interface can only be called up via IP using a browser (FireFox) for a relatively short time after I have rebooted the ESP via a power interruption. Later, or when I want to reload it after a hang, I always get a message that the IP is not available. >>> But I can ping the ESP successfully EVERY TIME?!?? That shouldn't really be possible then either, should it?!??

OK, today I can't seem to reach it at all via the webUI. Not even after a reset via power interruption. Pinging works without any problems, but in the browser either timeout or connection was reset by the server is displayed.

Unfortunately, I cannot see in the menu of the FritzBox whether the ESP is present, as the test additional repeater is not from AVM, but from Allnet. However, the MAC address is displayed in its menu.

Does anyone happen to know this behaviour? Anyone have an idea what I'm doing wrong? 

Many thanks (and I will now go down to the basement again, dismantle the aligned camera again to read out the SD card and the log file and post it here).

Timo

Translated with www.DeepL.com/Translator (free version)

**Comment by timosust** [comment]
OK, scheint als sei die SD Karte fehlerbehaftet. Im Rechner zeigt er mir das hier bei den Logs an und ich kann sie auch nicht öffnen.

OK, it seems that the SD card is faulty. In the computer it shows me this in the logs and I can't open them either.

![photo_2022-11-06_12-56-57](https://user-images.githubusercontent.com/115658413/200169383-d2929837-02f2-44c6-bf0e-f2a0cbe47204.jpg)

**Comment by jomjol** [comment]
Du kannst auch gerne in Deutsch schreiben.

**Comment by timosust** [comment]
OK, jetzt hab ich mal die SD Karte auf 1GB verkleinert, habe irgendwo gelesen, das das ne Ursache sein kann. Kann jetzt im SetUp max einmal ein Bild machen, dann geht nichts mehr. Auch nachdem ich wieder auf die vollen 4GB gegangen bin.

Er ist dann teilweise auch nicht per Ping erreichbar.

Habe beim wieder umstellen auf 4GB mal den detailierten Log in der config.ini aktiviert:

> 2022-11-06T15:01:24: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-06T15:01:24: =============================================================================================
2022-11-06T15:01:24: =================================== Main Started ============================================
2022-11-06T15:01:24: =============================================================================================
2022-11-06T15:01:24: Branch: 'HEAD', Tag: 'v12.0.1', Revision: dfe2c46, Date/Time: 2022-09-29 17:13
2022-11-06T15:01:30: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-06T15:04:55: download_get_handler
2022-11-06T15:04:56: download_get_handler
2022-11-06T15:06:59: 	handler_editflow - Start
2022-11-06T15:06:59: 	handler_editflow - Done


Ich habe jetzt noch gelesen, das der verbaute CH340 Chip wohl Probleme machen würde. Bestelle nachher mal ein FT232RL FTDI USB Seriell Konverter https://www.amazon.de/gp/product/B075N82CDL/ref=ox_sc_act_title_2?smid=A3MCS35JD5QFPQ&psc=1

Oder habt Ihr Empfehlungen für einen funktionierenden? Der hier https://randomnerdtutorials.com/esp32-cam-troubleshooting-guide/ empfohlene von AZ Delivery beim großen A hat bei selbigem halt einige schlechte Bewertungen und vor allem soll es nicht der Original FT232RL Chip sein. https://www.amazon.de/dp/B01N9RZK6I/ref=twister_B07ZQF5QYH?_encoding=UTF8&psc=1

Und nochmal 2 SanDisk Class 4 4GB Micro SD Karten.

Langsam wird das Projekt teuer *ggg*

Einen ESP32-Cam Chip hab ich noch in neu. Möchte den aber nicht auch jetzt gleich Himmeln *ggg* 

Stand jetzt ist also, ich kann nicht mal mehr das Set Up durchlaufen. Nochmal neu flashen geht aktuell nicht, weil die Micro USB Buchse abgebrochen ist und ich auf den Konverter warten muss. 

Das ist übrigens der PSRAM, wenn ich den richtigen Chip erwischt habe. Vielleicht hilft euch das weiter

![photo_2022-11-06_17-23-44](https://user-images.githubusercontent.com/115658413/200182349-49539fb0-4125-4ffb-b87b-5928516fa92f.jpg)

**Comment by timosust** [comment]
Habe jetzt nochmal auf den GND neben dem 5V Pin umgelötet, ob es vielleicht daran liegt. 

Reference Image kann ich Speichern, dann die Alignments markieren, und wenn ich auf save all to config.ini klicke, dann passiert nix mehr, außer das die Schaltflächen blockiert sind und ich auch nicht mehr die Seite neu laden kann.

Folgender Log wird ausgegeben:

> 2022-11-06T18:39:50: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-06T18:39:50: =============================================================================================
2022-11-06T18:39:50: =================================== Main Started ============================================
2022-11-06T18:39:50: =============================================================================================
2022-11-06T18:39:50: Branch: 'HEAD', Tag: 'v12.0.1', Revision: dfe2c46, Date/Time: 2022-09-29 17:13
2022-11-06T18:39:56: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-06T18:41:45: download_get_handler
2022-11-06T18:41:45: download_get_handler
2022-11-06T18:42:11: 	handler_editflow - Start
2022-11-06T18:42:11: 	handler_editflow - Done
2022-11-06T18:42:24: 	handler_editflow - Start
2022-11-06T18:42:24: 	handler_editflow - Done
2022-11-06T18:42:37: 	handler_editflow - Start
2022-11-06T18:42:37: 	handler_editflow - Done
2022-11-06T18:43:11: 	handler_editflow - Start
2022-11-06T18:43:11: 	handler_editflow - Done
2022-11-06T18:43:22: delete_post_handler
2022-11-06T18:43:22: download_get_handler
2022-11-06T18:43:23: upload_post_handler
2022-11-06T18:43:23: download_get_handler
2022-11-06T18:43:23: delete_post_handler
2022-11-06T18:43:23: download_get_handler
2022-11-06T18:43:24: upload_post_handler
2022-11-06T18:43:24: download_get_handler
2022-11-06T18:43:24: download_get_handler
2022-11-06T18:43:28: download_get_handler
2022-11-06T18:43:28: download_get_handler
2022-11-06T18:43:34: download_get_handler
2022-11-06T18:43:35: download_get_handler
2022-11-06T18:43:35: delete_post_handler
2022-11-06T18:43:35: 	handler_editflow - Start
2022-11-06T18:43:35: 	handler_editflow - Done
2022-11-06T18:43:35: delete_post_handler
2022-11-06T18:43:35: 	handler_editflow - Start
2022-11-06T18:43:35: 	handler_editflow - Done
2022-11-06T18:43:35: delete_post_handler
2022-11-06T18:43:35: 	handler_editflow - Start
2022-11-06T18:43:36: 	handler_editflow - Done
2022-11-06T18:43:36: delete_post_handler
2022-11-06T18:43:36: 	handler_editflow - Start
2022-11-06T18:43:36: 	handler_editflow - Done
2022-11-06T18:43:36: download_get_handler
2022-11-06T18:43:36: download_get_handler
2022-11-06T18:43:47: 	handler_editflow - Start
2022-11-06T18:43:48: 	handler_editflow - Done
2022-11-06T18:43:48: 	handler_editflow - Start
2022-11-06T18:43:48: 	handler_editflow - Done
2022-11-06T18:43:48: download_get_handler
2022-11-06T18:43:48: download_get_handler
2022-11-06T18:43:51: download_get_handler
2022-11-06T18:43:51: download_get_handler
2022-11-06T18:43:57: 	handler_editflow - Start
2022-11-06T18:43:57: 	handler_editflow - Done
2022-11-06T18:43:58: 	handler_editflow - Start
2022-11-06T18:43:58: 	handler_editflow - Done
2022-11-06T18:43:58: download_get_handler
2022-11-06T18:43:58: download_get_handler
2022-11-06T18:44:00: 	handler_editflow - Start
2022-11-06T18:44:01: 	handler_editflow - Done
2022-11-06T18:44:01: download_get_handler
2022-11-06T18:44:01: download_get_handler
2022-11-06T18:44:04: delete_post_handler
2022-11-06T18:44:04: download_get_handler
2022-11-06T18:44:04: 	handler_editflow - Start
2022-11-06T18:44:04: 	handler_editflow - Done
2022-11-06T18:44:04: delete_post_handler
2022-11-06T18:44:05: download_get_handler
2022-11-06T18:44:05: 	handler_editflow - Start
2022-11-06T18:44:05: 	handler_editflow - Done
2022-11-06T18:44:05: delete_post_handler
2022-11-06T18:44:05: download_get_handler

**Comment by timosust** [comment]
Hmmm, 

also mit dem bestellten UART Adapter bekomme ich ihn nicht geflasht bzw überhaupt in den Flash Mode. 

Ich hab das doch richtig verstanden. Zum Flashen muss GPI0 auf GND verbunden werden u verbunden bleiben?

Bei den gekauften ESP Cams ist ja der Adapter fest verbaut. Da bin ich in den Flash Mode gekommen indem ich RST u Flash gleichzeitig gedrückt u gehalten habe, ehrlich gesagt wild gedrückt bis er anfing blau zu blinken, dann habe ich losgelassen.

Ich habe jetzt nochmal diesen hier gekauft https://www.amazon.de/gp/product/B01N9RZK6I/ref=ewc_pr_img_1?smid=A1X7QLRQH87QA3&psc=1

Und wenn das nicht klappt, bestelle ich ganz vielleicht nochmal günstige ESPs beim schnellen Ali. 

Und dann wird das Projekt wohl begraben. Vermutlich bin ich einfach zu doof dafür. Trotz der super tollen Anleitung. Habe halt bisher keine Erfahrung mit Boardbasteleien Flashen und co. Ist bei mir aber komischerweise häufiger so, mache Schritt für Schritt alles genau nach Anleitung, und trotzdem machen Geräte nicht das was sich machen sollten.

**Reply by caco3** [reply]
> Bei den gekauften ESP Cams ist ja der Adapter fest verbaut. Da bin ich in den Flash Mode gekommen indem ich RST u Flash gleichzeitig gedrückt u gehalten habe

du must den flash button einen moment länger halten als den reset button. der flash button wird erst während dem aufstarten ausgewertet!

**Comment by caco3** [comment]
nimm eine variante mit ESP32-CAM-MB,da must du nichts verkabeln/löten

**Reply by timosust** [reply]
Das ist dann die Variante mit dem Adapter zum draufstecken? Prinzipiell ist das Verdrahten für mich kein Problem. Bin Elektriker. Nur beim SMD löten bin ich raus. Da zitter ich zu sehr 😉

Hatte ja extra welche mit eingebautem Adapter genommen. Wollte halt mal den eingebauten Adapter als Fehlerquelle ausschließen, weil das irgendwo stand. 
Werde nachher mal eine ältere Version testen.

**Reply by caco3** [reply]
ich habe auch noch solche mit eingebautem USB. Komischerweise schlägt da bei allen jedes OTA fehl.

Zum umlöten auf eine externe  Antenne kommst Du leider nicht ums SMD Löten drum herum 😶

**Reply by timosust** [reply]
Zum umlöten habe ich zum Glück schon jemand organisiert, der das beruflich macht ;-) 

Habe aber gezwungenermaßen hier im Waschkeller noch nen Repeater hinsetzen müssen. Da reicht jetzt auch die Interne. Mal schauen, wenn alles geht probiere ich es nochmal über die externe. evtl muss ich die mit nem SMA Kabel verlängern, damit ich an ne Stelle mit Empfang komme

Ich habe ja auch die mit eingebautem USB. Leider scheint die Buchse sehr empfindlich. Vorher, Flash per USB Kabel und WebInstaller ging, ebenso besagter Log. Jetzt wollte ich im Keller vor Ort mit der neuen Karte mitloggen..... Öffnet er zwar das Logfenster, zeigt aber nix mehr an?!?!??

**Comment by mad2xlc** [comment]
Ich habe ähnliche Probleme mit der aktuellen 12er Version gehabt. Ständige Timeouts im Webinterface. Ein Wechsel auf eine ältere Version hat das Problem wieder behoben.

**Reply by steff75** [reply]
Ich bekam es mit dem letzten Stable Release auch nicht anständig zum Laufen, auf 2 Geräten. Ständig hat es gehangen.
Dann habe ich einfach einen der letzten Rolling Releases genommen und es funktionierte auf Anhieb.

**Reply by mad2xlc** [reply]
Ja die laufen schon sehr gut :-)  Freue mich schon auf den nächsten Release.

**Comment by timosust** [comment]
Moin,

also, gerade nochmal mit dem FTDI Adapter probiert, dank dem neuen Webinstaller habe ich jetzt ein USB Log. Ich bekomme die Firmware über den Web Installer zwar nicht installiert, aber konnte jetzt den einen ESP bei der Einrichtung loggen. Nachdem ich das neue Referenzbild gespeichert habe, und die Allignments am einrichten bin, bricht er mit diesem


>  E (339699) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
> E (339699) task_wdt:  - IDLE (CPU 0)
> E (339699) task_wdt: Tasks currently running:
> E (339699) task_wdt: CPU 0: httpd
> E (339699) task_wdt: CPU 1: IDLE
> E (339699) task_wdt: Print CPU 0 (current core) backtrace
> 
> 
> Backtrace:0x400E9C74:0x3FFB09C0 0x40083B39:0x3FFB09E0 0x4000BFED:0x3FFF2CC0 0x40092A7D:0x3FFF2CD0 0x40090551:0x3FFF2CF0 0x400E8F5D:0x3FFF2D30 0x400E9871:0x3FFF2D50 0x40224B96:0x3FFF2DA0 0x4011D847:0x3FFF2DC0 0x4011D8FD:0x3FFF2E10 0x4011D96E:0x3FFF2E80 0x400D515F:0x3FFF2EB0 0x400D501D:0x3FFF2ED0 0x400D6D86:0x3FFF2EF0 0x400D8B66:0x3FFF2F30 0x4012030E:0x3FFF2F60 0x4008B682:0x3FFF2F80 0x40089D0D:0x3FFF2FA0 0x40089D9C:0x3FFF2FC0 0x4008A323:0x3FFF2FF0 0x40209093:0x3FFF3020 0x402090F5:0x3FFF3060 0x400F4E2E:0x3FFF3080 0x402244C6:0x3FFF30A0 0x400F7C9D:0x3FFF30D0 0x400F7D32:0x3FFF3100 0x400F7DD3:0x3FFF3140 0x400FE8B3:0x3FFF31C0 0x40196F37:0x3FFF31F0 0x4010028F:0x3FFF3230 0x400EBA49:0x3FFF3420 0x400EAD70:0x3FFF3460 0x400EB136:0x3FFF3510 0x400EA284:0x3FFF3530 
> 
> E (339699) task_wdt: Print CPU 1 backtrace
> 
> 
> Backtrace:0x40088019:0x3FFB0FC0 0x40083B39:0x3FFB0FE0 0x4000BFED:0x3FFC4F40 0x40092A7D:0x3FFC4F50 0x400E9E93:0x3FFC4F70 0x400E9E9F:0x3FFC4FA0 0x400E9EC6:0x3FFC4FC0 0x40090B20:0x3FFC4FE0 
> 
> E (344699) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
> E (344699) task_wdt:  - IDLE (CPU 0)
> E (344699) task_wdt: Tasks currently running:
> E (344699) task_wdt: CPU 0: httpd
> E (344699) task_wdt: CPU 1: IDLE
> E (344699) task_wdt: Print CPU 0 (current core) backtrace
> 
> 
> Backtrace:0x400E9C74:0x3FFB09C0 0x40083B39:0x3FFB09E0 0x4000BFED:0x3FFF2CC0 0x40092A7D:0x3FFF2CD0 0x400905B2:0x3FFF2CF0 0x400E8F5D:0x3FFF2D30 0x400E9871:0x3FFF2D50 0x40224B96:0x3FFF2DA0 0x4011D847:0x3FFF2DC0 0x4011D8FD:0x3FFF2E10 0x4011D96E:0x3FFF2E80 0x400D515F:0x3FFF2EB0 0x400D501D:0x3FFF2ED0 0x400D6D86:0x3FFF2EF0 0x400D8B66:0x3FFF2F30 0x4012030E:0x3FFF2F60 0x4008B682:0x3FFF2F80 0x40089D0D:0x3FFF2FA0 0x40089D9C:0x3FFF2FC0 0x4008A323:0x3FFF2FF0 0x40209093:0x3FFF3020 0x402090F5:0x3FFF3060 0x400F4E2E:0x3FFF3080 0x402244C6:0x3FFF30A0 0x400F7C9D:0x3FFF30D0 0x400F7D32:0x3FFF3100 0x400F7DD3:0x3FFF3140 0x400FE8B3:0x3FFF31C0 0x40196F37:0x3FFF31F0 0x4010028F:0x3FFF3230 0x400EBA49:0x3FFF3420 0x400EAD70:0x3FFF3460 0x400EB136:0x3FFF3510 0x400EA284:0x3FFF3530 
> 
> E (344699) task_wdt: Print CPU 1 backtrace
> 
> 
> Backtrace:0x40088019:0x3FFB0FC0 0x40083B39:0x3FFB0FE0 0x4000BFED:0x3FFC4F40 0x40092A7D:0x3FFC4F50 0x400E9E93:0x3FFC4F70 0x400E9E9F:0x3FFC4FA0 0x400E9EC6:0x3FFC4FC0 0x40090B20:0x3FFC4FE0 
> 
> E (349699) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
> E (349699) task_wdt:  - IDLE (CPU 0)
> E (349699) task_wdt: Tasks currently running:
> E (349699) task_wdt: CPU 0: httpd
> E (349699) task_wdt: CPU 1: IDLE
> E (349699) task_wdt: Print CPU 0 (current core) backtrace
> 
> 
> Backtrace:0x400E9C74:0x3FFB09C0 0x40083B39:0x3FFB09E0 0x4008808D:0x3FFF2CD0 0x400905BB:0x3FFF2CF0 0x400E8F5D:0x3FFF2D30 0x400E9871:0x3FFF2D50 0x40224B96:0x3FFF2DA0 0x4011D847:0x3FFF2DC0 0x4011D8FD:0x3FFF2E10 0x4011D96E:0x3FFF2E80 0x400D515F:0x3FFF2EB0 0x400D501D:0x3FFF2ED0 0x400D6D86:0x3FFF2EF0 0x400D8B66:0x3FFF2F30 0x4012030E:0x3FFF2F60 0x4008B682:0x3FFF2F80 0x40089D0D:0x3FFF2FA0 0x40089D9C:0x3FFF2FC0 0x4008A323:0x3FFF2FF0 0x40209093:0x3FFF3020 0x402090F5:0x3FFF3060 0x400F4E2E:0x3FFF3080 0x402244C6:0x3FFF30A0 0x400F7C9D:0x3FFF30D0 0x400F7D32:0x3FFF3100 0x400F7DD3:0x3FFF3140 0x400FE8B3:0x3FFF31C0 0x40196F37:0x3FFF31F0 0x4010028F:0x3FFF3230 0x400EBA49:0x3FFF3420 0x400EAD70:0x3FFF3460 0x400EB136:0x3FFF3510 0x400EA284:0x3FFF3530 
> 
> E (349699) task_wdt: Print CPU 1 backtrace

```



In Dauerschleife ab?!??

Hier der ganze Log
[esp-web-tools-logs (1).txt](https://github.com/jomjol/AI-on-the-edge-device/files/9991105/esp-web-tools-logs.1.txt)

**Comment by caco3** [comment, marked as answer]
hast Du es mal mit einer anderen SD-Karte versucht?

der webserver (`httpd` im Log) scheint ein Problem damit zu haben!

**Reply by timosust** [reply]
Bin ich gerade dran. sieht tendenziell gut aus. Allerdings ist mir gerade noch das ganze Fritz Mesh abgekackt... 

LIFT OFF, we have a Lift off.

Zumindest die Bedienung der Weboberfläche tut jetzt, und er analysiert auch Bilder. Muss aber erst noch weiter ausrichten u kalibrieren für taugliche Ergebnisse. Wobei er gerade schon richtig erkannt hat
 
Also war es womöglich tatsächlich die SD Karte.

Ich teste weiter

**Comment by timosust** [comment]
Moin,

ok, anscheinend lief er bis heute früh. Gegen 10 dann wieder komplett nicht erreichbar, aber ebenfalls der Repeater an dem er hängt. Sprich es scheint momentan jetzt noch an meinem Netzwerk zu hängen. 

Anfangs hat er gut erkannt, später dann wohl nicht mehr, aber das liegt sehr sicher an bisher grottigem leicht unscharfen Bild voll Reflektionen und grottig eingestellten ROIs. Dafür war ich dennoch über die gute Erkennungsrate überrascht.

ARRRGHH, doof wenn die Mietwohnung im 2. OG ist und der Zähler im KG. Trotz AVM Mesh Repeater bei der Nachbarin im EG mit ca 5-8m Luftlinie reicht dessen Abdeckung nicht aus. Fachwerk Lehmbau sei dank. Und der Allnet Repeater scheint sich nicht so recht mit dem Fritz Mesh zu vertragen. Lässt komischerweise auch kein Internet durch. 

Bleibt jetzt die große Frage, auf gut Glück nochmal 30€ in nen Fritz Repeater investieren, in der Hoffnung, dass es mit dem besser klappt. 

Oder deutlich mehr in ein Powerline System, wo die Wahrscheinlichkeit höher ist. Gehen würde das, WM u TR sind bei mir oben in der UV abgenommen. Phasengleichheit herstellen ist für mich das kleinste Problem.

Oder warten, bis die Stadtwerke mir hier innerhalb des nächsten halben Jahres Glasfaser kostenlos bis in die Whg legen und dabei dann fragen, ob ich nen Netzwerkkabel beilegen darf. Der Weg vom KG ins OG ist ja dann nicht mehr mein Problem ;-)

Ich schließe mal den Fall hier und hoffe das es dabei bleibt. Ganz vielen Dank für die Hilfe.

Also, Lösung war vermutlich eine neue SD Karte, welche ich neu, ohne vorher zu formatieren, beschrieben habe. Warum auch immer, es war der gleiche Hersteller mit dem gleichen Modell Intenso 4Gb Class4

**Reply by caco3** [reply]
> Also, Lösung war vermutlich eine neue SD Karte, welche ich neu, ohne vorher zu formatieren, beschrieben habe. 

Ev. hast Du sie falsch formatiert, und der ESP kann es halbwegs handlen?
Siehe https://github.com/jomjol/AI-on-the-edge-device/wiki/Installation#bangbang-attention-bangbang
